docker-compose.dev.yml

```diff
@@ -30,12 +30,12 @@ services:
 
   redis:
     image: redis:alpine
-    command: [ "redis-server", "--save", "3600", "10" ]
+    command: ["redis-server", "--save", "3600", "10"]
     volumes:
       - redis-data:/data
 
   ingest-file:
-    image: ghcr.io/alephdata/ingest-file:3.19.3
```

Comment:
I wonder if we should have a final ingest-file release before we bump this in Aleph?